server/src/resolvers/BuddyTask.js

```diff
@@ -0,0 +1,7 @@
+function newbie(parent, args, context) {
+  return context.prisma.buddyTask({ id: parent.id }).newbie()
+}
+
+module.exports = {
+  newbie,
+}
```

Comment:
Okay, I see that's there's no linting support added here and we have inconsistency with semicolons

```
module.exports = {
  newbie,
}
```
and in `Buddy.js` we have

https://github.com/BuildIt-Poland/buddy-app/blob/8eb6dec06d461425d12cc1e37a49348002ff517c/server/src/resolvers/Buddy.js#L13

Not a blocker for me, just wanted to point it out. I guess It can be added in other PR.